### PR TITLE
Support system scope in v3 auth

### PR DESCRIPTION
**Pull request by zhsj**
For #1907

https://docs.openstack.org/api-ref/identity/v3/?expanded=password-authentication-with-scoped-authorization-detail#system-scoped-example

https://github.com/openstack/keystone/blob/f5c9b094af8442219e75265a87551873e058003b/keystone/auth/schema.py#L97-L102

### CODE REVIEW

File: openstack/identity/v3/tokens/testing/requests_test.go [hunk 1]
```diff
@@ -248,6 +248,31 @@ func TestCreateProjectNameAndDomainIDScope(t *testing.T) {
...
+					"system": {
+						"all": true
+					}
+				}
+			}
+		}
+	`)
+}
+
+func TestCreateSystemScope(t *testing.T) {
```
Comment (jtopjian) [inline openstack/identity/v3/tokens/testing/requests_test.go]: @zhsj I think you want the body of the above test in this function?

**Comment by jtopjian** [comment]
@zhsj Thank you for submitting this. For future reference, we don't use the API docs as the only reference - we also need the actual OpenStack service code that implements the API feature. I've modified this PR description for you to include it.

File: openstack/identity/v3/tokens/testing/requests_test.go (hunk 1 above)
Comment (zhsj) [reply openstack/identity/v3/tokens/testing/requests_test.go]: I copied the `TestCreateProjectNameAndDomainNameScope` to `TestCreateSystemScope`, but I ended up modifying the `TestCreateProjectNameAndDomainNameScope` func...

Now fixed.

**Comment by jtopjian** [comment]
The OpenLab error is ignorable.

**Pull request merged**